ui-v2/tests/helpers/measure.js

```diff
@@ -0,0 +1,5 @@
+/* eslint no-console: "off" */
+import getMeasure from 'consul-ui/tests/lib/measure/getMeasure';
+let report;
+let len = 1;
+export default getMeasure(len, report);
```

Comment:
`len` here is the amount of times to run a test in order to then take an average for results, when I switch in a different reporter I switch this up to a higher number, I could have commented this in a little more, but I'm imagining this is a first draft of the reporting stuff so it's subject to change. `report` is empty just because for the moment I'm just adding in a custom reporter here when I'm interested, likely to move this to something more 'inject-y' at a later date.